internal/utils/test.go

```diff
@@ -105,3 +105,22 @@ func GetDataSourcesLength(state *terraform.State, tfVarName, dataSourceName stri
 
 	return instanceState, numAttribute, nil
 }
+
+func CheckRole(role string, scopeType string) bool {
+	var OrganizationRoles = []string{string(iam.ORGANIZATIONBILLINGADMIN), string(iam.ORGANIZATIONMEMBER), string(iam.ORGANIZATIONOWNER)}
+	var WorkspaceRoles = []string{string(iam.WORKSPACEACCESSOR), string(iam.WORKSPACEAUTHOR), string(iam.WORKSPACEMEMBER), string(iam.WORKSPACEOWNER), string(iam.WORKSPACEOPERATOR)}
+	var roles []string
+
+	if scopeType == "organization" {
+		roles = OrganizationRoles
+	} else if scopeType == "workspace" {
+		roles = WorkspaceRoles
+	}
```

Comment:
future proofing and just in case
```suggestion
        scopeType = strings.ToLower(scopeType)
	if scopeType == "organization" {
		roles = OrganizationRoles
	} else if scopeType == "workspace" {
		roles = WorkspaceRoles
	} else if scopeType == "deployment" {
	  return true
	}
```